app/Http/Controllers/CustomTourController.php

```diff
@@ -54,6 +50,32 @@ public function show(Request $request, $id)
         ]);
     }
 
+    public function pdfLayout(Request $request, $id)
+    {
+        $customTourItem = CustomTour::findOrFail($id);
+
+        $customTour = json_decode($customTourItem->tour_json, true);
```

Comment:
This can be accomplished automatically by setting a cast in the `CustomTour` model, for example:
```php
protected $casts = ['tour_json' => 'array'];
```

See https://laravel.com/docs/10.x/eloquent-mutators#array-and-json-casting

Reply:
This has been addressed.